_layouts/default.html

```diff
@@ -32,10 +32,6 @@
         <link rel="stylesheet" href="https://unpkg.com/animate.css@3.7.2/animate.min.css" integrity="sha512-doJrC/ocU8VGVRx3O9981+2aYUn3fuWVWvqLi1U+tA2MWVzsw+NVKq1PrENF03M+TYBP92PnYUlXFH1ZW0FpLw==" crossorigin="anonymous">
         <!-- Hero area slider css-->
         <link rel="stylesheet" href="/css/slider.css">
-        <!-- owl craousel css -->
-        <link rel="stylesheet" href="https://unpkg.com/owl.carousel@2.3.4/dist/assets/owl.carousel.min.css" integrity="sha512-tS3S5qG0BlhnQROyJXvNjeEM4UpMXHrQfTGmbQ1gKmelCxlSEBUaxhRBj/EFTzpbP4RVSrpEikbmdJobCvhE3g==" crossorigin="anonymous">
-        <link rel="stylesheet" href="https://unpkg.com/owl.carousel@2.3.4/dist/assets/owl.theme.default.min.css" integrity="sha512-sMXtMNL1zRzolHYKEujM2AqCLUR9F2C4/05cdbxjjLSRvMQIciEPCQZo++nk7go3BtSuK9kfa/s+a4f4i5pLkw==" crossorigin="anonymous">
-        <link rel="stylesheet" href="https://unpkg.com/jquery.fancybox@2.1.5/source/jquery.fancybox.css" integrity="sha512-Q3Vew98bTOHNPS36lEFNZ4Bf8VvFo90mRRftC6BJrkSps6uKUmcCGFDWQaYyn2j5j/QbLizpF6EmwTf59UnZ0A==" crossorigin="anonymous">
```

Comment:
Fancybox is still used (`_layouts/resources.html`) and this needs to be left in (the grouping in HTML is unfortunate though as it looks related to owl)